app/lib/helpers/ccbcSummaryGenerateFormData.tsx

```diff
@@ -295,13 +335,28 @@ const getFormDataFromApplication = (
   economicRegions,
   regionalDistricts
 ) => {
+  const communities = getCommunitiesTemplateNine(
+    applicationData?.applicationFormTemplate9DataByApplicationId?.nodes[0]
+      ?.jsonData?.geoNames
+  );
+  const templateNineSource = handleTemplateNineSource(
+    applicationData?.applicationFormTemplate9DataByApplicationId?.nodes[0]
+      ?.source
+  );
   return {
     formData: {
       counts: {
-        communities: null,
-        benefitingCommunities: null,
-        indigenousCommunities: null,
-        nonIndigenousCommunities: null,
+        communities:
+          (applicationData?.applicationFormTemplate9DataByApplicationId
+            ?.nodes[0]?.jsonData?.communitiesToBeServed || 0) +
```

Comment:
If, for whatever reason, there is no communitiesToBeServed do we want it to be 0 or just be null? Small thing we can address later though

Reply:
Yeah, I think that is a good point, since it is defaulting to 0 to avoid weird JS summation. I've created [NDT-579] to handle it if needed.